src/tests/tablet/homeUI/HomePageInformation.java

```diff
@@ -0,0 +1,135 @@
+package tests.tablet.homeUI;
+
+import static framework.utils.TimeManager.getCurrentDate;
+import static framework.utils.TimeManager.differenceBetweenTimes;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.utils.TimeManager;
+
+/**
+ * This TCs verify that the information displayed is correct in the home page when it is loaded.
+ * TC N�: 27 -> Room display name.
+ * TC N�: 28 -> Room code.
+ * TC N�: 33 -> Current time.
+ * TC N�: 19 -> Available message in {Now} tile.
+ * TC N�: 29 -> Color of {Now} tile.
+ * TC N�: 17 -> End of day message in {Next} tile.
+ * TC N�: 05 -> Time left display the time remaining form current time to midnight.
+ * @author Eliana Navia
+ * 
+ */
+public class HomePageInformation {
+
+	/**
+	 * [EN] This test case verifies that room display name is displayed 
+	 * in the main container window.
+	 * TC N�: 27
```

Comment:
the order in Test Case tittle should be
TC27: This test case verifies that room display name is displayed in the main container window.

correct that in all of your TC's (Team Conventions)